packages/circuits/tests/test-circuits/jwt-verifier-with-anon-email-domains-test.circom

```diff
@@ -0,0 +1,5 @@
+pragma circom 2.1.6;
+
+include "../../jwt-verifier-template.circom";
+
+component main { public [ pubkey ] } = JWTVerifier(121, 17, 1024, 128, 896, 72, 605, 1, 2);
```

Comment:
The `anonymousDomainsTreeRoot ` value should be public input here.

Reply:
Besides, the `pubKey` should not be public here; otherwise, this value can reveal the corresponding domain.